test/release/examples/primers/modules.chpl

```diff
@@ -171,6 +201,23 @@ module MainModule {
       writeln(bazBarFoo);
     }
 
+    /* Similarly, ``import`` statements also apply to the entire scope in which
+       they are defined.  Even if the ``import`` statement occurs after code
+       which would directly refer to its symbols, these references are still
+       valid.
+
+       Thus, the following declaration of ``bazBarFoo`` will store the result of
+       calling ``modToUse.baz`` on ``modToUse.bar`` and ``modToUse.foo``, which
+       is again ``28``.
```

Comment:
I think you could just summarize these two paragraphs here. The behaviour is identical to `use` as described right above. I think they are too close to repeat